crates/ruma-events/src/room/name.rs

```diff
@@ -11,51 +13,71 @@ pub type NameEvent = StateEvent<NameEventContent>;
 /// The payload for `NameEvent`.
 #[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
 #[ruma_event(type = "m.room.name", kind = State)]
+#[non_exhaustive]
```

Comment:
```suggestion
#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
```

Unless there is a reason to not have this configurable?